app/styles/spacing.scss

```diff
@@ -2,6 +2,7 @@
 
 $space-map: (
   "none": $space-none,
+  "px": 1px,
```

Comment:
Do we still need? Might be useful at some point I guess

Reply:
My preference is to not have stuff we aren't really using, unless it's part of a larger system. But up to you :)